packages/fast-form-generator-react/src/form/form-item.array.style.ts

```diff
@@ -20,59 +18,81 @@ const styles: ComponentStyles<FormItemArrayClassNameContract, {}> = {
-        ...applyPopupHeadingStyles(),
-        "& button": {
-            lineHeight: "1",
-            fontSize: toPx(14),
-            cursor: "pointer",
-            background: "transparent",
-            border: "none",
-            padding: `${toPx(4)}`,
-            "&:focus": {
-                outline: "none",
-            },
-        },
+        position: "relative",
+        display: "flex",
+        flexDirection: "column",
     },
-    formItemArray_actionMenu: {
-        ...applyCleanListStyle(),
-        ...applyAriaHiddenStyles(),
-        ...applyPopupMenuStyles(),
-        "& $formItemArray_actionMenuItem__add, & $formItemArray_actionMenuItem__remove": {
-            ...localizePadding(12, 12, 12, 36),
-            width: "100%",
-            ...ellipsis(),
-            textAlign: "left",
-            color: colors.black,
-            "&::before": {
-                position: "absolute",
-                content: "''",
-                opacity: ".6",
-                pointerEvents: "none",
-                width: toPx(16),
-                height: toPx(16),
-                left: toPx(10),
-            },
-            "&:hover": {
-                backgroundColor: colors.grayBackground,
-            },
-        },
+    formItemArray_control: {
+        width: "100%",
+        verticalAlign: "middle",
+        marginTop: toPx(12),
     },
-    formItemArray_actionMenuItem__add: {
+    formItemArray_controlAddButton: {
+        position: "absolute",
+        right: "0",
+        top: "12px",
+        appearance: "none",
+        background: "none",
+        border: "none",
+        margin: "8px",
+        width: "20px",
+        height: "16px",
+        zIndex: "1",
+        borderRadius: "2px",
+        "&:focus": {
+            ...insetStrongBoxShadow(colors.pink),
+            outline: "none",
+        },
         "&::before": {
+            position: "absolute",
+            content: "''",
+            opacity: ".6",
+            pointerEvents: "none",
+            width: toPx(16),
+            height: toPx(16),
+            left: toPx(2),
+            top: "0",
             background: plus,
         },
     },
-    formItemArray_actionMenuItem__remove: {
+    formItemArray_controlLabel: {
+        ...applyLabelStyle(),
+        ...applyControl(),
+        verticalAlign: "middle",
+        lineHeight: "32px",
+    },
+    formItemArray_existingItemList: {
+        ...applyCleanListStyle(),
+        "& li": {},
+    },
+    formItemArray_existingItemListItem: {
+        position: "relative",
+        height: "36px",
+        lineHeight: "36px",
+        paddingLeft: "26px",
         "&::before": {
-            background: minus,
+            position: "absolute",
+            content: "''",
+            opacity: ".6",
+            pointerEvents: "none",
+            top: toPx(11),
+            width: toPx(16),
+            height: toPx(16),
+            background: lines,
+            left: "0",
+        },
+        "& a": {
```

Comment:
@janechu do we not have a class for this? Would be *ideal* if we didn't have to target elements wholesale.

Reply:
We are very precisely targeting the `<a>` inside of the `<li>` which has the class `formItemArray_existingItemListItem`, there should only be a single `<a>` inside.

I can add a class and use that instead, however I would argue the reasoning is that it is more maintainable, I don't think it's a good argument that the targeting is "wholesale" in this case.

Reply:
Cool - primary consideration was the reasoning. Thx!